### PR TITLE
Lower linalg_vector_norm

### PR DESCRIPTION
**Pull request by cowanmeg**
Fixes https://github.com/pytorch/xla/issues/4492

---
Lower linalg_vector_norm and adds unit tests.

### CODE REVIEW

File: test/cpp/test_aten_xla_tensor.cpp [hunk 1]
```diff
@@ -950,6 +950,71 @@ TEST_F(AtenXlaTensorTest, TestLinalgSVD) {
   ExpectCounterChanged("xla::_linalg_svd", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestLinalgVectorNorm) {
```
Comment (wonjoolee95) [inline test/cpp/test_aten_xla_tensor.cpp:953]: nit: can we add the counters check to verify that the ops are getting properly dispatched to XLA? Example:
```
ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
ExpectCounterChanged("xla::linalg_vector_norm", cpp_test::GetIgnoredCounters());
```

Comment (cowanmeg) [reply test/cpp/test_aten_xla_tensor.cpp:953]: thanks!

Comment (wonjoolee95) [reply test/cpp/test_aten_xla_tensor.cpp:953]: Let's add it for all the other `linalg_vector_norm` unit tests below as well

**Comment by wonjoolee95** [comment]
Seems like CircleCi is down, jobs not starting -- https://status.circleci.com 😢

**Review by wonjoolee95 — APPROVED** [review]
LGTM. Thanks, Meghan!

**Pull request merged**